src/compat/browser_compatibility_types.ts

```diff
@@ -104,15 +103,10 @@ const MediaSource_ : typeof MediaSource|undefined =
                                   win.MSMediaSource;
 /* tslint:enable no-unsafe-any */
 
-const MediaKeys_ : ICompatMediaKeysConstructor|undefined = (() => {
+const MediaKeys_ : ICompatMediaKeysConstructor = (() => {
   /* tslint:disable no-unsafe-any */
-  if (shouldUseWebKitMediaKeys()) {
-    return win.WebKitMediaKeys;
-  }
   return win.MediaKeys != null ? win.MediaKeys :
-         win.MSMediaKeys != null ? win.MSMediaKeys :
          win.MozMediaKeys != null ? win.MozMediaKeys :
-         win.WebKitMediaKeys != null ? win.WebKitMediaKeys :
```

Comment:
I understand why both `MSMediaKeys` and `WebKitMediaKeys` lines were removed, but are we sure that there is no case left where we would not go into the corresponding IE11 or Webkit EME compatibility code and where having `MediaKeys_` equal to one of them would still be needed?

Reply:
I'm a bit uncomfortable with this idea mainly for one reason. If we decide to use one of MS of WebKit API without going in the compatibility code, there are high chances that the code would crash precisely by the lack of compatibility (non-existing APIs).

Actually, i would find it safer to decide to be compatible just by the presence of the API.
For exemple, for WebKitMediaKeys, there are much more chances that the API would go into the same behavior between a Safari and another webkit based browser, no ?

Reply:
OK, I understand for the first part.
So, you prefer to throw `"MEDIA_KEYS_NOT_SUPPORTED"` in those cases (if they exist) than a un-debuggable error,

I'm not sure if I follow with the second part of your comment, though. For me it has no relation with repeating the same behavior on Webkit-based browsers. It would just be a fallback in cases where the webkit logic we just added would not be triggered but there would still be a `WebKitMediaKeys` in window.